front/app/api/groups/useUpdateGroup.ts

```diff
@@ -0,0 +1,30 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import groupKeys from './keys';
+import { IGroup, IGroupUpdate } from './types';
+import streams from 'utils/streams';
+import { API_PATH } from 'containers/App/constants';
```

Comment:
Personally don't see this as a real issue, but just notice that the imports are not organized as usual.